src/win/getaddrinfo.c

```diff
@@ -219,9 +217,7 @@ void uv_freeaddrinfo(struct addrinfo* ai) {
   char* alloc_ptr = (char*)ai;
 
   /* release copied result memory */
-  if (alloc_ptr != NULL) {
-    uv__free(alloc_ptr);
-  }
+  uv__free(alloc_ptr);
```

Comment:
This isn't really applicable here. alloc_ptr is the local cast of a pointer passed in which is immediately lost. We would need to pass in a `struct addrinfo** ai` to actually nullify it.